### YamlMime:ManagedReference
items:
- uid: System.Runtime.Remoting.RemotingException
  id: RemotingException
  children:
  - System.Runtime.Remoting.RemotingException.#ctor
  - System.Runtime.Remoting.RemotingException.#ctor(System.String)
  - System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RemotingException
  nameWithType: RemotingException
  fullName: System.Runtime.Remoting.RemotingException
  type: Class
  summary: "Undantag som utlöses när något är fel under fjärrkommunikation."
  remarks: "RemotingException använder HRESULT COR_E_REMOTING, som har värdet 0x8013150B.       RemotingException använder förvalet <xref:System.Object.Equals%2A>-implementering som stöder referenslikheten.</xref:System.Object.Equals%2A>       En lista med inledande egenskapsvärden för en instans av RemotingException finns RemotingException konstruktorer."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class RemotingException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor
  id: '#ctor'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException()
  nameWithType: RemotingException.RemotingException()
  fullName: System.Runtime.Remoting.RemotingException.RemotingException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initierar en ny instans av den <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klass med standardegenskaper."
  remarks: "<xref:System.Runtime.Remoting.RemotingException>ärver från en <xref:System.SystemException>klass.</xref:System.SystemException></xref:System.Runtime.Remoting.RemotingException> I följande tabell visas första egenskapsvärden för en instans av klassen <xref:System.Runtime.Remoting.RemotingException>: | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`|   | <xref:System.Exception.Message%2A>| En tom sträng (&quot;&quot;) |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException ();
    parameters: []
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(String)
  nameWithType: RemotingException.RemotingException(String)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initierar en ny instans av den <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klass med ett angivet meddelande."
  remarks: "<xref:System.Runtime.Remoting.RemotingException>ärver från en <xref:System.SystemException>klass.</xref:System.SystemException></xref:System.Runtime.Remoting.RemotingException> I följande tabell visas första egenskapsvärden för en instans av klassen <xref:System.Runtime.Remoting.RemotingException>: | Egenskapstypen | Villkoret |   |-------------------|---------------|   | <xref:System.Exception.InnerException%2A>| `null`|   | <xref:System.Exception.Message%2A>| Den `message` strängen |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Felmeddelandet som förklarar varför undantaget inträffade."
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(SerializationInfo,StreamingContext)
  nameWithType: RemotingException.RemotingException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initierar en ny instans av den <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klass från serialiserade data."
  remarks: "Den här konstruktorn anropas under deserialisering till att fylla Undantagsobjektet som överförs via en dataström. (Mer information om serialisering finns [XML och SOAP-serialiseringen](~/add/includes/ajax-current-ext-md.md).)"
  syntax:
    content: protected RemotingException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Det objekt som innehåller objektdata som serialiserade."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Detaljerad information om käll- och målservrar för undantaget."
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "The <code>info</code> parameter is <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>."
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(String,Exception)
  nameWithType: RemotingException.RemotingException(String,Exception)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initierar en ny instans av den <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klass med ett angivet meddelande och en referens till det interna undantaget som är orsaken till det här undantaget."
  remarks: "Ett undantag som utlöses när ett direkt resultat av ett tidigare undantag innehåller en referens till det tidigare undantaget i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>-egenskap returnerar samma värde som överförs till konstruktorn eller `null` om den <xref:System.Exception.InnerException%2A>egenskapen tillhandahåller inte värdet i det interna undantaget för konstruktorn.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       I följande tabell visas första egenskapsvärden för en instans av klassen <xref:System.Runtime.Remoting.RemotingException>: | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Referens för inre undantag. |   | <xref:System.Exception.Message%2A>| Meddelandet felsträng. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException (string message, Exception InnerException);
    parameters:
    - id: message
      type: System.String
      description: "Felmeddelandet som förklarar varför undantaget inträffade."
    - id: InnerException
      type: System.Exception
      description: "Undantag som är orsaken till det aktuella undantaget. Om den `InnerException` parametern är inte <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, det aktuella undantaget utlöses i en <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> block som hanterar det interna undantaget."
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.ArgumentNullException
  isExternal: true
  name: System.ArgumentNullException
- uid: System.Runtime.Remoting.RemotingException.#ctor
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException()
  nameWithType: RemotingException.RemotingException()
  fullName: System.Runtime.Remoting.RemotingException.RemotingException()
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(String)
  nameWithType: RemotingException.RemotingException(String)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(SerializationInfo,StreamingContext)
  nameWithType: RemotingException.RemotingException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(String,Exception)
  nameWithType: RemotingException.RemotingException(String,Exception)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Runtime.Remoting.RemotingException.#ctor*
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException
  nameWithType: RemotingException.RemotingException
